swarm/network/kademlia.go

```diff
@@ -367,12 +371,15 @@ func (k *Kademlia) EachBin(base []byte, pof pot.Pof, o int, eachBinFunc func(con
 	kadDepth := depthForPot(k.conns, k.MinProxBinSize, k.base)
 
 	k.conns.EachBin(base, pof, o, func(po, size int, f func(func(val pot.Val, i int) bool) bool) bool {
-		if startPo > 0 && endPo != k.MaxProxDisplay {
-			startPo = endPo + 1
-		}
+		//if the peer's bin is smaller than the kademlia depth,
```

Comment:
but now this PR really needs to wait for the new depth calculation

Reply:
agreed

Reply:
Please do _not_ use "smaller". Please review the terminology introduced here:

https://github.com/ethersphere/go-ethereum/pull/1051

If you don't agree with the terms, let's discuss and settle on something everyone can agree with.